.jenkins/dsl/jobs.groovy

```diff
@@ -18,6 +18,27 @@ def getDefaultJobParams(String repoName = 'optaplanner') {
             repository: repoName,
             credentials: "${GIT_AUTHOR_CREDENTIALS_ID}",
             token_credentials: "${GIT_AUTHOR_TOKEN_CREDENTIALS_ID}"
+        ],
+        env: [:],
+        pr: [:]
+    ]
+}
+
+Map getMultijobPRConfig() {
+    return [
+        jobs : [
+            [
+                id: 'Tests',
+                primary: true,
+            ], [
+                id: 'Apps',
+                waitForId: 'Tests',
+                repository: 'kogito-apps'
```

Comment:
Is there a way of defining how to build the downstream repository? For example, building the entire kogito-apps may be wasteful, as it is only the task-assigning module of kogito-apps, which brings the dependency on optaplanner there.

Reply:
For now, downstream projects are not configurable ... We could make it possible via env configuration and profile launch.